src/rabbit_core_metrics.erl

```diff
@@ -384,3 +389,34 @@ get_gen_server2_stats(Pid) ->
         [] ->
             not_found
     end.
+
+auth_attempt_succeeded(Ip, Username) ->
+    %% {Id, Total, Succeeded, Failed}
```

Comment:
Did you mean `Ip` instead of `Id`?

Should we use `RemoteIp` or `RemoteHost` here as well? See https://github.com/rabbitmq/rabbitmq-server/pull/2436#pullrequestreview-490800812